search/tasks.py

```diff
@@ -136,3 +137,11 @@ def populate_query_memberships(percolate_query_id):
         percolate_query_id (int): Database id for the PercolateQuery to populate
     """
     api.populate_query_memberships(percolate_query_id)
+
+
+@app.task
```

Comment:
So we can ensure this task doesn't get dropped.

```suggestion
@app.task(acks_late=True)
```